Tools/TestWebKitAPI/cocoa/DragAndDropSimulator.h

```diff
@@ -146,4 +146,11 @@ typedef NSDictionary<NSNumber *, NSValue *> *ProgressToCGPointValueMap;
 
 @end
 
+@interface DragAndDropSimulator (DOMElementDrag)
+
+- (void)runFromElement:(NSString *)startSelector toElement:(NSString *)endSelector;
+
+@end
+
```

Comment:
Extra newline here. I'll remove it before asking to `merge-queue` this PR.

Reply:
Removed in https://github.com/WebKit/WebKit/commit/d9f8d6940b38b36eafdacb26de28cd7d1d62a331